### DISCUSSION TITLE
Customized pairwise comparisons when using trans_diff

### DISCUSSION THREAD

**Discussion by eperezv** [ChiLiubio/microeco#217] [Q&A]
Hello,
I am interested in performing differential abundance analysis with microeco, and would like to know if there is a way to restrict the pairwise comparisons (e.g. using wilcox) in any of the methods to some specific treatment pairs. I have several groups and I am only interested in the comparisons with the control. I was wondering is there is any way of doing that, I read the manual but did not find it.
Thank you.

**Comment by ChiLiubio** [comment]
Hi. Currently, there is no parameters in trans_diff that can directly control the selection of paired groups as paired groups input is generally complicated for users, which is usually list format. In my viewpoint, to do what you described, it is feasible to split dataset into subgroups with paired or more groups to do differential analysis (https://chiliubio.github.io/microeco_tutorial/basic-class.html#subset-of-samples), or select the interested tests by filtering the comparions in the result after performing all the test.
